lib/cmd/cmd.go

```diff
@@ -35,6 +35,7 @@ func Main(cmd *cobra.Command) {
 
 	if err != nil {
 		const errExitCode = 1
+		log.Error(ctx, "Command execution failed", err)
```

Comment:
this has been fixed in https://github.com/omni-network/omni/pull/2088, please remove this.